src/syntax/specification/specification.spec.ts

```diff
@@ -161,12 +161,22 @@ describe('Syntax Element Specification', () => {
         }
 
         test('register element specification entry and verify', () => {
-            registerElementSpecificationEntry('dummy0', {
+            const registerElement = registerElementSpecificationEntry('dummy0', {
                 label: 'dummy0',
                 type: 'Block',
                 category: 'dummy',
                 prototype: DummyElementBlock,
             });
+
+            expect(registerElement).toBeTruthy();
+            const registerElementAgain = registerElementSpecificationEntry('dummy0', {
+                label: 'dummy0',
+                type: 'Block',
+                category: 'dummy',
+                prototype: DummyElementBlock,
+            });
+            expect(registerElementAgain).toBeFalsy();
```

Comment:
put this in a separate test
`register duplicate element specification entry and verify`